components/dialog/harvester/SupportBundle.vue

```diff
@@ -8,7 +8,7 @@ import GraphCircle from '@/components/graph/Circle';
 import Banner from '@/components/Banner';
 
 export default {
-  name: 'SuppportBundle',
+  name: 'SupportBundle',
```

Comment:
api break?

Reply:
Looks good - I don't see the component being referenced by name with the misspelled name.